task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -49,19 +50,20 @@ spec:
         SKIP_MSG="We found 0 supported files"
         grep -q "$SKIP_MSG" stdout.txt || test_not_skipped=$?
 
-        HACBS_ERROR_OUTPUT=$(make_result_json -r ERROR)
-
         if [[ "$SNYK_EXIT_CODE" -eq 0 ]] || [[ "$SNYK_EXIT_CODE" -eq 1 ]]; then
           cat sast_snyk_check_out.json
           HACBS_TEST_OUTPUT=
           parse_hacbs_test_output $(context.task.name) sarif sast_snyk_check_out.json  || true
 
         # When the test is skipped, the "SNYK_EXIT_CODE" is 3 and it can also be 3 in some other situation
         elif [[ "$test_not_skipped" -eq 0 ]]; then
-          HACBS_ERROR_OUTPUT=$(make_result_json -r SKIPPED -t "$SKIP_MSG")
+          note="Task \`$(context.task.name)\` is skipped: Task is skipped because command \`snyk code test\` found 0 supported files."
```

Comment:
--> note="Task \`$(context.task.name)\` ~is~ skipped: \`snyk code test\` found zero supported files."

Reply:
Fixed.